feat: fuse-overlayfs configuration (#2685)
codenvy/che-docs: modules/end-user-guide/pages/accessing-fuse.adoc

:_content-type: PROCEDURE
:description: Accessing /dev/fuse from your workspaces.
:keywords: user-guide, fuse, overlay, device, /dev/fuse, pod, overrides
:navtitle: Accessing /dev/fuse
:page-aliases:

[id="accessing-fuse"]
= Accessing /dev/fuse from a workspace

> Using fuse-overlayfs requires access to `/dev/fuse`. This section describes the steps required for `/dev/fuse` to be accessible to workspace containers.

Reviewer: ```suggestion
You must have access to `/dev/fuse` to use fuse-overlayfs. This section describes how to make `/dev/fuse` accessible to workspace containers.
```

.Prerequisites

> * The administrator has enabled access to `/dev/fuse` by following xref:administration-guide:configuring-fuse.adoc[]

Reviewer: ```suggestion
* The administrator has enabled access to `/dev/fuse` by following xref:administration-guide:configuring-fuse.adoc[].
```

.Procedure

> . Determine a workspace to add the required annotations to. The required annotations defined in xref:administration-guide:configuring-fuse.adoc[] are as follows.

Reviewer: what does this step do? can we skip it since all it says is "select a workspace" (if I understand it correctly)? maybe this can be a prerequisite instead.

Author: This step is for the user to choose any of their workspaces to use fuse-overlayfs with. I updated this PR to have it as a prerequisite, WDYT?

![Screenshot from 2024-02-20 18-11-05](https://github.com/eclipse-che/che-docs/assets/83611742/0bdd5a4b-63f2-4eae-9ee0-88dc73788067)
